src/main/java/org/openbase/bco/bcozy/util/Groups.java

```diff
@@ -63,5 +84,33 @@ public String toString(UnitConfigType.UnitConfig object) {
         };
     }
 
+    public static void removeGroup(UnitConfigType.UnitConfig group) throws InterruptedException {
+        try {
+            Registries.getUserRegistry().removeAuthorizationGroupConfig(group);
+        } catch (CouldNotPerformException ex) {
+            ExceptionPrinter.printHistory(ex, LOGGER);
+        }
+    }
 
+    public static boolean addGroup(String groupName) throws InterruptedException {
```

Comment:
Do not use boolean return values for exception handling. throw a CouldNotPerformException instead.

Reply:
this method is only used to encapsulate the thrown exception, it uses the "tryAddGroup" function that does throw the CouldNotPerformeException in case of errors.